test/rest/status.test.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+define(['shared_helper', 'chai'], function (helper, chai) {
+  var rest;
+  var utils = helper.Utils;
+  var expect = chai.expect;
+
+  // RSL8
+  describe('rest/status', function () {
```

Comment:
Optional - maybe test can be updated to check for channel status values

Reply:
I think it would take a lot of effort for us to test this on the ably-js side and really it's something the realtime team should be testing themselves so I'm gonna opt to not do this one for now